<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LXQtFancyMenuConfiguration</class>
 <widget class="QDialog" name="LXQtFancyMenuConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fancy Menu settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="generalGB">
     <property name="title">
      <string>General</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="iconCB">
        <property name="text">
         <string>Icon:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="iconLE">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="iconPB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="showTextCB">
        <property name="text">
         <string>Button text:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="textLE">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="customFontCB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Custom font size:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="customFontSizeSB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> pt</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>11</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="menuGB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Menu file</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="menuFileL">
        <property name="text">
         <string>Menu file:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="menuFilePathLE"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="chooseMenuFilePB"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Keyboard Shortcut</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="ShortcutSelector" name="shortcutEd">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Click the button to record shortcut:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="searchGB">
     <property name="title">
      <string>Search</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="filterClearCB">
        <property name="text">
         <string>Clear search upon showing menu</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="popupGB">
     <property name="title">
      <string>Popup</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="1">
       <widget class="QComboBox" name="buttRowPosCB"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="buttRowPosLabel">
        <property name="text">
         <string>Buttons row position</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="categoryViewPosLabel">
        <property name="text">
         <string>Categories position</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="categoryViewPosCB"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShortcutSelector</class>
   <extends>QToolButton</extends>
   <header location="global">LXQtGlobalKeysUi/ShortcutSelector</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>customFontCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>customFontSizeSB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showTextCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLE</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iconCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconLE</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iconCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconPB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
